### DISCUSSION TITLE
is Bootstrap Vue using bootstrap version 5?

### DISCUSSION THREAD

**Discussion by JenuelDev** [bootstrap-vue/bootstrap-vue#6541] [Q&A] [answered]
is Bootstrap Vue using bootstrap version 5?

**Comment by Hiws** [comment, marked as answer]
No, BootstrapVue does not support Bootstrap 5 yet.
You can subscribe to this issue to stay updated: https://github.com/bootstrap-vue/bootstrap-vue/issues/5507

**Comment by stevecj** [comment]
Regardless of specific support for Bootstrap 5, do we know what (if any) incompatibilities there are trying to use BV with Bootstrap 5? Perhaps, BV happens to not run into any of the breaking changes?

I did an experiment using BV pagination with the latest MDBootstrap (based on B5) and that worked fine.

**Reply by stevecj** [reply]
If I run into any conflicts with breaking changes, I was thinking I could do the following. Are there any flaws with this idea?

1. Within my project, make a copy of the component that has the conflict
2. Edit the template in the copy to use B5-compatible markup.
2. Make copies of any BV component(s) my project uses that incorporate the one that I copied & modified, changing their imports to refer to the updated copy.
